zppy/templates/e3sm_diags.bash

```diff
@@ -155,6 +155,15 @@ echo
 
 # Prepare configuration file
 cat > e3sm.py << EOF
+# Path information:
+# For climo, data is copied from these directories to a local directory
+# climo_dir_source_lnd_obs = ${climo_dir_source_lnd_obs}
+# climo_dir_source_lnd_ref = ${climo_dir_source_lnd_ref}
+# climo_dir_source_atm_obs = ${climo_dir_source_atm_obs}
+# climo_dir_source_atm_ref = ${climo_dir_source_atm_ref}
+# climo_diurnal_dir_source_obs = ${climo_diurnal_dir_source_obs}
+# climo_diurnal_dir_source_ref = ${climo_diurnal_dir_source_ref}
+# For ts, data is used directly from original directories
```

Comment:
I think in bash an undefined variable will just show up as an empty string, so this shouldn't cause problems